Maestro/subscriptions.json

```diff
@@ -366,6 +396,33 @@
         }
       }
     },
+    // Update dependencies in CoreFX feature/utf8string
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/feature/utf8string/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/feature/utf8string/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master/Latest.txt"
```

Comment:
(No change needed, just some thoughts on including `standard/release/2.0.0/Latest.txt`, `projectk-tfs/master/Latest.txt`, etc. triggers:)

Triggers only affects the cadence of auto-update PRs being generated, not their contents. (The [`RemoteDependencyBuildInfo`s](https://github.com/dotnet/corefx/blob/815e529644a876e027dc62f95855dd6332e4ccff/dependencies.props#L80-L115) configure what updates happen. Someone will actually have to update [`DependencyBranch`](https://github.com/dotnet/corefx/blob/815e529644a876e027dc62f95855dd6332e4ccff/dependencies.props#L75) while making sure the `projectk-tfs`, `buildtools` etc. BuildInfos still point at `master` because those aren't branching.)

For `master`, that means triggers and dependencies should be in sync all the time, so none of the dependencies go out of date if the few marked as triggers don't build in a while.

For a feature branch, I think it's fine to limit the triggers to what people working in the branch care about. It reduces noise a little, and the branch has to merge up with `master` at some point in the future anyway.

Reply:
> limit the triggers to what people working in the branch care about

Yup, that was my thought since these branches should not have a lifetime beyond however long until they are merged to master.